tests/test_openclhpp.cpp

```diff
@@ -4214,10 +4214,10 @@ void clSVMFree_stubForMemoryAllocation(cl_context context, void *svm_pointer,
     int cmock_num_calls)
 {
     TEST_ASSERT_EQUAL_PTR(svm_pointer, testMemory);
-    delete svm_pointer; 
+    delete (int*) svm_pointer; 
 }
 void testSVMMemoryAllocation()
-{
+{//
```

Comment:
```suggestion
{
```

Something committed accidentally?